File: 30-days-of-code/python/day_2.py
#!/bin/python3

import math
import os
import random
import re
import sys

# Complete the solve function below.
def solve(meal_cost, tip_percent, tax_percent):
    # This functions takes in meal_cost, tip_percent
    # and tax_percent and returns the total price rounded
    # to the nearest dollar
    return meal_cost * (1 + (tip_percent + tax_percent)/100)

if __name__ == '__main__':
    meal_cost = float(input())
    tip_percent = int(input())
    tax_percent = int(input())
    total_cost = solve(meal_cost, tip_percent, tax_percent)
    print(round(total_cost))

File: 30-days-of-code/scala/day_2.scala
import scala.math

object Solution {
    // Complete the solve function below.
    def solve(meal_cost: Double, tip_percent: Int, tax_percent: Int) {
        var totalCost: Double = ((100 + tip_percent.toDouble + tax_percent.toDouble) / 100) * meal_cost
        totalCost = Math.round(totalCost)
        println(totalCost.toInt)
    }

    def main(args: Array[String]) {
        val stdin = scala.io.StdIn
        val meal_cost = stdin.readLine.trim.toDouble
        val tip_percent = stdin.readLine.trim.toInt
        val tax_percent = stdin.readLine.trim.toInt

        solve(meal_cost, tip_percent, tax_percent)
    }
}